src/app/style/abstract/_media.scss

```diff
@@ -63,6 +63,12 @@
     }
 }
 
+@mixin largest-screen {
+    @media (min-width: 1920px) {
+        @content;
+    }
+}
```

Comment:
Make sure that you really need this here, and if yes - call it `after-desktop` or something like that, to stick to the existing methodology.